test/benchmark/benchmark.jl

```diff
@@ -4,8 +4,8 @@ using Random
 Random.seed!(10)
 
 println("Forest fire model")
-include("../forest_fire_defs.jl")
-a = @benchmark data=step!(forest, dummystep, forest_step!, 100, Dict(:status => [length, count]), when=1:100) setup=(forest = model_initiation(f=0.05, d=0.8, p=0.01, griddims=(20, 20), seed=2))
+include("forestFire_defs.jl")
+a = @benchmark data=run!(forest, dummystep, forest_step!, 100, agent_properties=[(:status, length), (:status, count)], when=1:100) setup=(forest = model_initiation(f=0.05, d=0.8, p=0.01, griddims=(20, 20), seed=2))
```

Comment:
`agent_properties` should be made into `adata`, etc. Otherwise warnings are thrown which slow code down (if someone were to run these examples in their computer`